tracer/src/Datadog.Trace/AspNet/SharedItems.cs

```diff
@@ -54,6 +54,21 @@ internal static Scope TryPopScope(HttpContext context, string key)
 
             return default(Scope);
         }
+
+        internal static Scope TryPeakScope(HttpContext context, string key)
```

Comment:
```suggestion
        internal static Scope TryPeekScope(HttpContext context, string key)
```

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2495/commits/6785f56e8af6d600e01818116fa36b76c7e0cdb6